src/line_printer.cc

```diff
@@ -40,6 +43,12 @@ LinePrinter::LinePrinter() : have_blank_line_(true), console_locked_(false) {
   console_ = GetStdHandle(STD_OUTPUT_HANDLE);
   CONSOLE_SCREEN_BUFFER_INFO csbi;
   smart_terminal_ = GetConsoleScreenBufferInfo(console_, &csbi);
+  
+  // Try enabling ANSI escape sequence support on Windows 10 terminals.
+  DWORD mode;
+  if (GetConsoleMode(console_, &mode)) {
```

Comment:
Do we even want to call this even when `smart_terminal_` is `false`?

Reply:
Ah, I missed that completely. Fixed and rebased.